tests/dummy/app/templates/elements/article.hbs

```diff
@@ -1,11 +1,3 @@
-<article class='post' id='post_3'>
-This is an article for John Doe.
-</article>
-
-<article class='post' id='post_1'>
-This is an article for Jane Doe.
-</article>
-
-<article class='post' id='post_2'>
-This is another article for Jane Doe.
-</article>
+<article class="post" id="post_3"> This is an article for John Doe.</article>
+<article class="post" id="post_1"> This is an article for Jane Doe.</article>
+<article class="post" id="post_2"> This is another article for Jane Doe.</article>
```

Comment:
If there is only text inside a element then 1 line is perfect. Say you need a span inside a `<h1>` then it would be multiple lines.